packages/ui/v2/core/Shell.tsx

```diff
@@ -141,6 +142,8 @@ const Layout = (props: LayoutProps) => {
         <Toaster position="bottom-right" />
       </div>
 
+      <TeamsUpgradeBanner />
```

Comment:
Maybe later we can make a `DynamicBanners` component that will auto display banners according to each specific User/System status/etc... cc @Udit-takkar

Reply:
I think this was the plan - we just weren't 100% sure the best approach to handle the data. If I remember correctly a similar thing to how ToastContext works and just have the ability to push into and render all banners.

Reply:
Yeah @Udit-takkar  is already thinking about this. My idea was to create a separate table in the DB where we push all these notifications to, so we have a single place where all of them live, and then have one root notification banner which pulls all of them from the DB on every page, and shows them. Also see the thread in Threads about this for more opinions 😄